packages/react/src/forms/ControlGroup/ControlGroup.module.css

```diff
@@ -0,0 +1,65 @@
+.ControlGroup__container {
+  display: flex;
+  flex-direction: column;
+  gap: var(--base-size-8);
+  font-family: var(--brand-body-fontFamily);
+  font-size: var(--brand-text-size-100);
+  border: none;
+  margin: 0;
+  padding: 0;
+}
+
+.ControlGroup__label {
+  font-weight: var(--base-text-weight-semibold);
+  margin: 0;
+  padding: 0;
+  margin-bottom: var(--base-size-8);
+}
+
+.ControlGroup__caption {
+  font-size: var(--brand-text-size-100);
+  letter-spacing: var(--brand-text-letterSpacing-100);
+  line-height: var(--brand-text-lineHeight-100);
+  color: var(--brand-color-text-muted);
+}
+
+.ControlGroup__validation {
+  display: flex;
+  gap: var(--base-size-4);
+  font-size: var(--brand-text-size-100);
+  font-weight: var(--base-text-weight-regular);
+  line-height: var(--brand-text-lineHeight-100);
```

Comment:
This is quite a bit smaller than the usual FormControl validation, does it need to be? That one is 16px vs 14px here. Would recommend we keep that the same for consistency.

FormControl:
<img width="719" alt="Screenshot 2024-11-27 at 13 43 01" src="https://github.com/user-attachments/assets/2f85771f-ef7f-42bc-8543-5a558fc65a14">

Reply:
There's a bit more context on this in my other PR here which makes the change across the rest of PB.

https://github.com/primer/brand/pull/831

TL;DR is that our FormControl validation size didn't match Figma, so after chatting to Dani we've agreed to bring it down in the library to keep it consistent. The same goes for the hint.

Reply:
Gotcha, good to know 👍